src/components/media_control/public/media-control.scss

```diff
@@ -323,12 +323,12 @@
           margin-right: 6px;
           opacity: 1;
           &:hover { opacity: 1; }
-          &:before { content:"\\e004"; }
+          &:before { content:"\e004"; }
```

Comment:
Selector should have depth of applicability no greater than 2, but was 5<br>Colon after property should be followed by one space